$OpenBSD: patch-binding_gyp,v 1.2 2012/12/05 00:45:05 abieber Exp $

Bad hack to get it to use correct implementation, since it is
recognizing the OpenBSD as Linux.

--- binding.gyp.orig	Sun Jun 24 05:39:12 2012
+++ binding.gyp	Wed Nov 28 09:28:50 2012
@@ -18,10 +18,10 @@
 					{
 						'cflags': ['-Wno-deprecated-declarations'],
 						'defines': ['USE_CORO'],
-						'ldflags': ['-pthread'],
+						'ldflags': ['-pthread -L${LOCALBASE}/lib'],
 					}
 				],
-				['OS == "linux" or OS == "solaris" or OS == "freebsd"', {'defines': ['CORO_UCONTEXT']}],
+				['OS == "linux" or OS == "solaris" or OS == "freebsd"', {'defines': ['CORO_ASM']}],
 				['OS == "mac"', {
 					'defines': ['CORO_SJLJ'],
 					 'xcode_settings': {
